static/scripts/rewards/web3/erc20-permit.ts

```diff
@@ -85,7 +84,7 @@ export async function checkPermitClaimable(permit: Erc20Permit, signer: ethers.p
     return false;
   }
 
-  const { balance, allowance } = await fetchTreasury(permit, provider);
+  const { balance, allowance } = await fetchTreasury(permit.permit.permitted.token, permit.owner, provider);
```

Comment:
permit.permit should have been adjusted.

Reply:
- https://github.com/ubiquity/pay.ubq.fi/pull/177/commits/b7483f785cfc652c09397dc746ed28fb3184fd87
I implemented the syntax change in the sync ERC pr